### PR TITLE
Added dotenv package to load environment variables from .env files. 

### PR DESCRIPTION
**Pull request by usithadev**
Many people add their environment vars in a `.env` file. In this project, to load `BOT_TOKEN` environment variable from `.env` file I added [dotenv](https://www.npmjs.com/package/dotenv) package.
Dotenv is a zero-dependency module that loads environment variables from a .env file into [process.env](https://nodejs.org/docs/latest/api/process.html#process_process_env).

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
Just saw this, can you see that CI is failing? I can't speak for @EdJoPaTo who is the maintainer here, but most likely it's a good idea if you fix up your changes so that CI passes :)

File: source/bot/index.ts
Comment (EdJoPaTo) [inline source/bot/index.ts]: I think naming it envconfig is a bit confusing. I would maybe reuse the name of the package `dotenv()` seems easier to get to me.

**Review by EdJoPaTo — CHANGES_REQUESTED** [review]
`xo` (the linter) does not want to mix up the imports with code. The function call should be moved below the imports.
Then its also at the same place where the BOT_TOKEN is read.

Also there are some other formatting nitpicks. (Run `npm test` or `npx xo --fix`.) Do you want to fix them or should I do that when I find the time?

File: source/bot/index.ts
Comment (usithadev) [reply source/bot/index.ts]: Renamed it.

File: .gitignore
Comment (EdJoPaTo) [inline .gitignore]: Sorting the entries there

```suggestion
.env
dist
node_modules
```

File: source/bot/index.ts
Comment (EdJoPaTo) [inline source/bot/index.ts]: Removing empty line  and sort the entries

```suggestion
import {Bot, session} from 'grammy';
import {config as dotenv} from 'dotenv';
```

File: source/bot/index.ts
Comment (EdJoPaTo) [inline source/bot/index.ts]: I think "load" is a better comment here than "configure" as nothing is really configured here?

```suggestion
dotenv(); // Load from .env file
```

**Review by EdJoPaTo — CHANGES_REQUESTED** [review]
I mostly had some nitpicks. Once they are solved I can merge this PR :sparkles:

**Review by EdJoPaTo — APPROVED** [review]
Awesome! Thank you for your contribution!

**Pull request merged**
